scripts/github_issue_retriever.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import argparse
+
+from bugbug import db, github
+from bugbug.utils import zstd_compress
+
+
+class Retriever(object):
+    def retrieve_issues(
+        self, owner=str, repo=str, state=str, retrieve_events=bool
+    ) -> None:
+        db.download(github.GITHUB_ISSUES_DB)
+        github.download_issues(owner, repo, state, retrieve_events)
```

Comment:
This script fetches all issues sorted by creation date (earliest -> latest).

For fetching new issues and updates I was thinking to use a similar approach as in https://github.com/mozilla/bugbug/blob/master/scripts/bug_retriever.py#L24 (get ids of issues that were modified since last run, delete them from the database and add them again during download process).

Github has `since` parameter, allowing to get updates after a given time https://docs.github.com/en/rest/reference/issues#list-issues-for-a-repository 

Wonder if I should address this in a separate PR (and after the db artifact is created) since `db.last_modified` is using the artifact url to get `Last-Modified` header.

Reply:
> Wonder if I should address this in a separate PR (and after the db artifact is created) since db.last_modified is using the artifact url to get Last-Modified header.

As you prefer, you can do it here or in a follow-up PR.
The script should work also when db.last_modified is unavailable (it is a mistake that bug_retriever doesn't handle that case, I will fix that), because in some cases we need to regenerate it (e.g. if we change the format) and when we do we start from scratch.

Reply:
Ok, thanks, I'll create a follow-up PR for that